tests/dashboards/verifyMongodbDashboards_test.js

```diff
@@ -1,43 +1,48 @@
 Feature('Test Dashboards inside the MongoDB Folder');
 
 Before(async ({ I }) => {
-  await I.Authorize('admin', 'pmm2023fortesting!');
+  await I.Authorize();
 });
 
 Scenario(
   'PMM-T305 Open the MongoDB Instance Summary Dashboard and verify Metrics are present and graphs are displayed @nightly @dashboards',
   async ({ I, dashboardPage }) => {
-    I.amOnPage(dashboardPage.mongodbOverviewDashboard.url);
+    const url = I.buildUrlWithParams(dashboardPage.mongodbOverviewDashboard.url, {
+      from: 'now-5m',
+      cluster: 'replicaset',
+    });
+
+    I.amOnPage(url);
     dashboardPage.waitForDashboardOpened();
     await dashboardPage.expandEachDashboardRow();
     await dashboardPage.verifyMetricsExistence(dashboardPage.mongodbOverviewDashboard.metrics);
-    await dashboardPage.verifyThereAreNoGraphsWithNA();
-    await dashboardPage.verifyThereAreNoGraphsWithoutData();
```

Comment:
Why to remove methods that verify that data is present? Same in the test bellow?